components/TextFields/src/MDCTextInputController.m

```diff
@@ -84,6 +84,8 @@
 }
 
 @interface MDCTextInputController () {
+  UIColor *_floatingPlaceholderColor;
+  UIColor *_inlinePlaceholderColor;
```

Comment:
In this case I've overridden both the setter and the getter so that's not true.
![screen shot 2017-06-19 at 1 18 07 pm](https://user-images.githubusercontent.com/1271525/27297180-de83a33e-54f1-11e7-847d-6508a2a9ea8c.png)